### DISCUSSION TITLE
How to cite MONAI in publication?

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#2292] [General]
Suppose I'm going to use MONAI in my research, how should I cite MONAI in my publication?

**Comment by function2-llx** [comment]
Is it okay just to mention I used MONAI in the text and append the github link in the footnote?

**Reply by Nic-Ma** [reply]
Hi @wyli and @ericspod ,

I think this a FAQ question, may I know where the doc or guide is?

Thanks.

**Reply by rijobro** [reply]
Maybe time to move on @tvercaut 's suggestion of creating a proper citation?

**Reply by wyli** [reply]
there is an issue on this topic https://github.com/Project-MONAI/MONAI/issues/1166

**Reply by function2-llx** [reply]
So there's a DOI available(http://doi.org/10.5281/zenodo.4323059)? Thanks!
